allensdk/brain_observatory/behavior/behavior_project_api/behavior_ophys_analysis_query_utils.py

```diff
@@ -0,0 +1,309 @@
+from pymongo import MongoClient
+import yaml
+import pandas as pd
+import numpy as np
+import json
+import os
+import glob
+import traceback
+import datetime
+
+import time
+from tqdm import tqdm as progress
+
+from IPython.core.display import display, HTML
+display(HTML("<style>.container { width:100% !important; }</style>"))
+
+from allensdk.brain_observatory.behavior import behavior_project_cache as bpc
+import allensdk.brain_observatory.behavior.cell_metrics as cm
+
+class Database(object):
+    '''
+    utilities for connecting to MongoDB databases (mouseseeks or visual_behavior_data)
+
+    parameter:
+      database: defines database to connect to. Can be 'mouseseeks' or 'visual_behavior_data'
+    '''
+
+    def __init__(self, database_server='visual_behavior_data', ):
+        # get database ip/port info from a text file on the network (maybe not a good idea to commit it)
+        db_info_filepath = '//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/mongo_db_info.yml'
```

Comment:
should parametrize and cross-platformize

Reply:
moved path and filename into init params (so my reply to the previous comment about additional params should have been 'yes'). 

Path tested in linux. Double forward slash should work on Windows as well, but not tested yet. Forward slashes in general work in Windows.